daemon/src/wallet.rs

```diff
@@ -286,6 +305,9 @@ impl xtra::Actor for Actor<ElectrumBlockchain> {
     async fn started(&mut self, ctx: &mut xtra::Context<Self>) {
         let this = ctx.address().expect("self to be alive");
 
+        this.send_async_safe(InitialSync)
+            .await
+            .expect("we just started");
         self.tasks.add(this.send_interval(SYNC_INTERVAL, || Sync));
```

Comment:
please be aware this might do `InitialSync` and `Sync` almost at the same time. perhaps we could move adding the task to `InitialSync` handler (or dispatch it synchronously, so the task is only added after the initial sync?).

Reply:
I know what you mean but I can't get it working. 
Afaik I can't use `this.send(...)` as this blocks the actor. Can you provide a patch please?

Reply:
I had a stab, and figured out that the easiest way would be to actually block in the `started()` waiting for the cache - if you deem this unacceptable, please drop the commit I added to this PR - note that the actual sync will be executed in the task.
I wrote in the description that we should be OK, as we don't expect anything to work without the cache anyway (so it's a part of the construction phase).

Reply:
that's a good idea. Did you try it and does it run?